docs/content/rn-intro-con.adoc

```diff
@@ -1,6 +1,11 @@
+[id='concept-explanation-{context}']
+= Overview of {PRODUCT}
 
-[id='rn-intro-con']
+{PRODUCT} enables your team to develop, build and deploy enterprise integration solutions on a private Kubernetes cluster managed by Red Hat. 
```

Comment:
I would suggest OpenShift Dedicated being used here rather than Kubernetes.  The description of an OpenShift Dedicated cluster is that it is running Kubernetes, but RHMI is describing at a layer above OpenShift Dedicated so should use that as the reference name.